src/platform/functions/index.js

```diff
@@ -1,103 +1,128 @@
 const functions = require('firebase-functions');
-const puppeteer = require('puppeteer');
+// const puppeteer = require('puppeteer');
 const admin = require('firebase-admin');
+const { Cluster } = require('puppeteer-cluster');
 
 admin.initializeApp();
 
 const timeout = (ms) => new Promise((resolve) => setTimeout(resolve, ms));
 
 exports.scrapeEventbrite = functions
   .runWith({
-    timeoutSeconds: 30,
-    memory: '1GB',
+    timeoutSeconds: 120,
+    memory: '2GB',
```

Comment:
let's not increase this
```suggestion
    timeoutSeconds: 60,
    memory: '1GB',
```

Let's only increase the timeoutseconds. I think 1 gb of ram should be enough